inc/class-core-sitemaps-registry.php

```diff
@@ -42,13 +42,13 @@ public function add_sitemap( $name, $provider ) {
 	}
 
 	/**
-	 * Returns a single sitemap provider.
+	 * Returns a single registered sitemaps.
```

Comment:
This should probably read either:

1. Returns a single registered sitemaps provider.
2. Returns a single registered sitemap.

Reply:
Done, I went with 1.